lib/basedriver/commands/execute-child.js

```diff
@@ -0,0 +1,128 @@
+import _ from 'lodash';
+import B from 'bluebird';
+import vm from 'vm';
+import log from '../logger';
+import { attach } from 'webdriverio';
+
+// duplicate defining these keys here so we don't need to re-load a huge appium
+// dependency tree into memory just to run a wdio script
+const W3C_ELEMENT_KEY = 'element-6066-11e4-a52e-4f735466cecf';
+const MJSONWP_ELEMENT_KEY = 'ELEMENT';
+
+async function runScript (driverOpts, script, timeout) {
```

Comment:
should timeout have a default value?

Reply:
it does, in the important place where the request from the user is handled (in `execute-child.js`) but i'll add a check to make sure it's included